hack/create.sh

```diff
@@ -46,6 +51,13 @@ if [[ $# -ne 1 ]]; then
     exit 1
 fi
 
+for f in $ARTIFACTS; do
+    if [[ ! -f $f ]]; then
+        echo error: $f must exist for fakeRF
```

Comment:
Instead of doing this here, write the code inside createorupdate.go

Reply:
I was thinking to even generate dummy certs if no certs are provided. So creation will not fail...